packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.tsx

```diff
@@ -69,7 +70,7 @@ export const DateRangeValidationMessage = ({
 
   return (
     <FieldMessage
-      id={id}
+      id={validationMessage.dateStart ? dateStartId : dateEndId}
```

Comment:
Instead of these ternaries, it might actually be clearer to have another early return instead. Otherwise you should apply the same ternary `message` and `status` (although I'd say it's less favourable than an if statement since there's 3).

```
if (validationMessage.dateStart) {
  return <FieldMessage id={dateStartId} ...etc />
}

return <FieldMessage id={dateEndId} ...etc />
```